### DISCUSSION TITLE
HAID from payload

### DISCUSSION THREAD

**Discussion by gu33mis** [alexkn/node-red-contrib-homeconnect#76] [Q&A] [answered]
Is it possible to get HAID for for the home connect request node from the payload? I would like to reuse some of the workflows for multiple appliances.

Thank you.

**Comment by alexkn** [comment, marked as answer]
If the haid in the node configuration is empty, it will be read from msg.haid.
https://github.com/alexkn/node-red-contrib-homeconnect/blob/25173945873c64465599a1b93d348db2d3c2f410/nodes/home-connect-request.js#L47
